stetho/src/main/java/com/facebook/stetho/inspector/elements/android/RealWindowDescriptor.java

```diff
@@ -0,0 +1,34 @@
+package com.facebook.stetho.inspector.elements.android;
+
+import android.view.View;
+
+import com.facebook.stetho.common.Accumulator;
+import com.facebook.stetho.inspector.elements.AbstractChainedDescriptor;
+import com.facebook.stetho.inspector.elements.android.window.WindowFetcher;
+import com.facebook.stetho.inspector.elements.android.window.WindowWrapper;
+
+import java.util.List;
+
+import static com.facebook.stetho.inspector.elements.android.ApplicationDescriptor.belongToActivity;
+
+
+public class RealWindowDescriptor extends AbstractChainedDescriptor<WindowWrapper> {
+	static {
+		WindowFetcher.hook();
```

Comment:
hook is already a concept in `AbstractChainedDescriptor`.  Strongly recommend avoiding using that name for something that's got a different lifecycle/meaning.